### ISSUE TITLE
Ability to package the files of the braille publication

### ISSUE THREAD

**Issue by avneeshsingh** [daisy/ebraille#33]
As a braille reader I would also like to read braille publication offline, in form of a packaged publication.

Problem statement: Though a significant amount of content is read online, the need of reading the content offline still persists. Distribution of multiple files for offline reading raises many challenges and it is also confusing for the readers to manage multiple files. Therefore, we need a way to package the multiple files of the braille publication.
The packaging should be an additional feature which producers can use, we should not make it a mandatory requirement in the EBraille specifications to ensure that both unpackaged set of files and packaged set of files are valid EBraille publications.

**Comment by jrbowden** [comment]
EPUB is basically a ZIP file containing the relevant files. Likewise Microsoft Word DOCX is a ZIP file containing the relevant files. Perhaps we should take the same approach?

**Comment by avneeshsingh** [comment]
@jrbowden Indeed I envision approach somewhat like EPUB. But EPUB is not a straight forward zip, one needs to index the files in proper order. So, one cannot zip a file, and rename it to .epub. In the technical analysis document we have listed different ways of achieving it and EPUB is one of them.

@nicolegaines if we are sure that users are consuming the ebraille offline then we can assume that packaging is required. But the world is changing, and online content is becoming more common. So, if we envision that we would also like to put ebraille content online, so that people can read it from website, then packaging becomes irrelevant.
This is why I proposed to have packaging as a separate specification. If one wants to use ebraille offline then packaging can be used, and if one wants to put ebraille online like a webpage, then packaging would not be used.
